.codacy.yaml

```diff
@@ -0,0 +1,11 @@
+exclude_paths:
+  - '.bundle/**'
+  - 'spec/**/*'
+  - 'benchmarks/**/*'
+  - '*.min.js'
+  - '**/tests/**'
+  - '**/functionaltests/**/*'
```

Comment:
Do we exclude the functional tests because of the same reason as in the back-end? If so, we might want to create an issue to change the functional tests - I know we have that kind of issue in back-end repo

Reply:
Changed to integration tests.